client/src/modules/Notifications/components/NotificationSettingsTable.tsx

```diff
@@ -50,14 +51,16 @@ function buildColumns(onEdit: (record: NotificationDto) => void, onDelete: (reco
             <a onClick={() => onEdit(record)}>Edit</a>
           </div>
           <div>
-            <Popconfirm
+            <CustomPopconfirm
               title="Are you sure to delete this notification?"
               onConfirm={() => onDelete(record)}
               okText="Yes"
               cancelText="No"
             >
-              <a>Delete</a>
-            </Popconfirm>
+              <a href="#" onClick={e => e.preventDefault()}>
+                Delete
+              </a>
```

Comment:
Here and in all places below:  
Similar to `CustomPopconfirm`you've recently made please create component to this kind of links in order to avoid duplication of this `href="#" onClick={e => e.preventDefault()}` everywhere